youtube_dl/extractor/ora.py

```diff
@@ -30,9 +29,9 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, display_id)
 
         video_data = self._search_regex(
-            r'"current"\s*:\s*({[^}]+?})', webpage, 'current video')
+            r'"video"\s*:\s*({[^}]+?})', webpage, 'current video')
         m3u8_url = self._search_regex(
-            r'"hls_stream"\s*:\s*"([^"]+)', video_data, 'm3u8 url', None)
+            r'hls_stream\s*:\s*"([^"]+)', video_data, 'm3u8 url', None)
```

Comment:
Both quoted and quoteless should be tried. Looking at their JSON they can switch back any moment.